src/theme/foundations/spacing.ts

```diff
@@ -106,6 +106,12 @@ const reservoirSpacingTokens = {
       py: reservoirSpacingValues["s"],
     },
   },
+  heading: {
```

Comment:
I was wondering whether this update should go into the spacing style guide but I realized we don't really show any semantic tokens there. Should we?

Reply:
Ah, yes, these additions should be added to the Spacing Style Guide. We do include semantic tokens there. 

![image](https://github.com/user-attachments/assets/705ba036-26de-4e87-86b4-d20c6c4601f7)

I also need to update the Design Tokens Development Guide.